doc/pveclibmaindox.h

```diff
@@ -756,6 +718,118 @@ vec_mul10uq (vui128_t a)
 *  \sa \ref i32_example_0_0_0
 *  \sa \ref i32_example_0_0_1
 *
+*  \subsubsection mainpage_sub3 pveclib is not a matrix math library
+*
+*  The pveclib does not implement general purpose matrix math operations.
+*  These should continue to be developed and improved within existing
+*  projects (ie LAPACK, OpenBLAS, ATLAS, etc).
+*  We believe that pveclib will be helpful to implementors of matrix
+*  math libraries by providing a higher level, more portable,
+*  and more consistent vector interface for the PowerISA.
+*
+*  The decision is still pending on;
```

Comment:
The list that follows "still pending on;"

When I ask if the LTC need these implementations, I am told that "oh someone is working on that". Usually to find out later that "someone" gave up (or was reassigned ...).

So the question for you is: should PVECLIB provide(crypto, compress/decompress, Small Vector library functions, etc. As examples for the general ecosystem. Even if openssl and libmvec already have implementations. 

Since I am:
- Only one guy
- Retired
- Not getting payed

There is no percentage in signing up for work, no one has asked for.

Reply:
That's not an easy question to answer, one retired guy. Examples are always good, but how many, how complex, and how much time to spend on them is a question. There are other things in far flung places that need attention, too...